src/style.css

```diff
@@ -0,0 +1,28 @@
+body {
+  margin: 0;
+}
+
+.container {
+  display: inline-block;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+}
```

Comment:
Also add here margin-right or margin left and background position

Reply:
In previous commit i did margin-left for all stars exept first, isn't that more correct?
And why i need to background position here?